specification/monitor/resource-manager/Microsoft.Monitor/preview/2023-10-01-preview/azuremonitor.json

```diff
@@ -0,0 +1,1980 @@
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ]
+    },
+    "PipelineGroupListResult": {
+      "type": "object",
+      "description": "The response of a PipelineGroup list operation.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "The PipelineGroup items on this page",
+          "items": {
+            "$ref": "#/definitions/PipelineGroup"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "format": "uri",
+          "description": "The link to the next page of items",
+          "readOnly": true
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "PipelineGroupProperties": {
+      "type": "object",
+      "description": "Properties that need to be specified to create a new pipeline group instance.",
+      "properties": {
+        "replicas": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Defines the amount of replicas of the pipeline group instance."
+        },
+        "receivers": {
+          "type": "array",
+          "description": "The receivers specified for a pipeline group instance.",
+          "items": {
+            "$ref": "#/definitions/Receiver"
+          },
+          "x-ms-identifiers": []
+        },
+        "processors": {
+          "type": "array",
+          "description": "The processors specified for a pipeline group instance.",
+          "items": {
+            "$ref": "#/definitions/Processor"
+          },
+          "x-ms-identifiers": []
+        },
+        "exporters": {
+          "type": "array",
+          "description": "The exporters specified for a pipeline group instance.",
+          "items": {
+            "$ref": "#/definitions/Exporter"
+          },
+          "x-ms-identifiers": []
+        },
+        "service": {
+          "$ref": "#/definitions/Service",
+          "description": "The service section for a given pipeline group instance."
+        },
+        "networkingConfigurations": {
+          "type": "array",
+          "description": "Networking configurations for the pipeline group instance.",
+          "items": {
+            "$ref": "#/definitions/NetworkingConfiguration"
+          },
+          "x-ms-identifiers": []
+        },
+        "provisioningState": {
+          "$ref": "#/definitions/ProvisioningState",
+          "description": "The provisioning state of a pipeline group instance. Set to Succeeded if everything is healthy.",
+          "readOnly": true
+        }
+      },
+      "required": [
+        "receivers",
+        "processors",
+        "exporters",
+        "service"
+      ]
+    },
+    "PipelineGroupUpdate": {
```

Comment:
>PipelineGroupUpdate [](http://example.com/codeflow?start=5&length=19)

can the same object PipelineGroup as PUT be used ?

Reply:
That is generated by typespec. I think it is in purpose to support future non-updatable attributes.